Shared/Mobile-Rounds.ViewModels/Regular/Region/RegionModel.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Mobile_Rounds.ViewModels.Shared;
```

Comment:
This class needs to be renamed. Once you  merge with `develop`, you will see that a `RegionModel` already exists, which is what Chris was suggesting. Please rename this to be something like `RegionModelSource` or something.